README.md

```diff
@@ -1,2 +1,249 @@
-# FOOOF.jl
+# PyFOOOF.jl
 Julia interface to [FOOOF](https://github.com/fooof-tools/fooof)
+
+[![Build Status][build-img]][build-url] [![CodeCov][codecov-img]][codecov-url]
+
+[build-img]: https://github.com/beacon-biosignals/PyFOOOF.jl/workflows/CI/badge.svg
+[build-url]: https://github.com/beacon-biosignals/PyFOOOF.jl/actions
+[codecov-img]: https://codecov.io/github/beacon-biosignals/PyFOOOF.jl/badge.svg?branch=main
+[codecov-url]: https://codecov.io/github/beacon-biosignals/PyFOOOF.jl?branch=main
+
+
+## Installation
+This package uses [`PyCall`](https://github.com/JuliaPy/PyCall.jl/) to make
+[FOOOF](https://fooof-tools.github.io/fooof/index.html) available from within Julia.
+Unsurprisingly, FOOOF and its dependencies need to be installed in order for this to work
+and PyFOOOF will attempt to install when the package is built.
+
+By default, this installation happens in the "global" path for the Python used
+by PyCall. If you're using PyCall via its hidden Miniconda install, your own
+Anaconda environment, or a Python virtual environment, this is what you want.
+(The "global" path is sandboxed to the Conda/virtual environment.) If you are
+however using system Python, then you should set `ENV["PIPFLAGS"] = "--user"`
+before `add`ing / `build`ing the package. By default, PyFOOOF will use the latest
+FOOOF 1.x release available on [PyPI](https://pypi.org/project/FOOOF/), but this can also
+be changed via the `ENV["FOOOFVERSION"] = version_number` for your preferred
+`version_number`.
+
+Note that FOOOF-Python uses [`matplotlib`](https://matplotlib.org/) for plotting, but does not install it automatically as a dependency.
```

Comment:
Doesn't PyPlot install matplotlib?

Reply:
This gets complicated .... but actually we don't need the PyPlot dependency for the core functionality, so I'll drop it and leave it up to the user how they want to handle it